README.md

```diff
@@ -10,16 +10,37 @@ alpinejs-devtools is a simple extension to help you debug [Alpine.js](https://gi
 
 ## Installation
 
-- [Get the Chrome Extension](https://chrome.google.com/webstore/detail/alpinejs-devtools/fopaemeedckajflibkpifppcankfmbhk)
-- [Get the Firefox Extension](https://addons.mozilla.org/firefox/addon/alpinejs-devtools/)
+-   [Get the Chrome Extension](https://chrome.google.com/webstore/detail/alpinejs-devtools/fopaemeedckajflibkpifppcankfmbhk)
```

Comment:
Any reason for the extra whitespace?

Reply:
Prettier adds the extra space

Reply:
Would be better without but maybe we should have linting/formatting in place 🤷🏼‍♀️